### PR TITLE
Add aarch64 and ppc64le

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: .ci_support/linux_aarch64_.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+cdt_arch:
+- aarch64
+cdt_name:
+- cos7
+channel_sources:
+- conda-forge,c4aarch64,defaults
+channel_targets:
+- conda-forge main
+ctng_binutils:
+- 2.29.1
```
Comment (isuruf) [inline .ci_support/linux_aarch64_.yaml]: @jjhelmus, what was the reason to choose 2.29.1 while it is 2.31.1 on all other platforms?

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: .ci_support/linux_aarch64_.yaml (hunk 1 above)
Comment (jjhelmus) [reply .ci_support/linux_aarch64_.yaml]: binutils >=2.30 require glibc 2.24 which is newer than the CentOS 7 target of 2.17, see:
https://github.com/crosstool-ng/crosstool-ng/commit/36bb675a71cc66a2abd69bc82d613f2153d3791f

**Comment by isuruf** [comment]
I'm going to merge this as is and maybe look at aarch64 later.

**Pull request merged**